data/korath/korath outfits.txt

```diff
@@ -105,7 +131,7 @@ outfit "Systems Core (Small)"
 	"hull repair rate" 0.2
 	"hull energy" 0.2
 	"hull heat" 0.3
-	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small number of "hull-walking" welding robots that can make minor repairs to the ship's outer armor while in flight.`
+	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`
```

Comment:
Technically, yes, but practically, no.  Usually when we say "small number" we mean something something easily counted by a human, these are thousands of sand-grain sized robots that would be more convenient to measure by volume than absolute quantity, and "small amount" implies this better.

Reply:
(also if anyone is wondering and not privy to the discussion on Discord, there were so many different descriptions and even names for the bots the Korath use, that I decided to standardise them on how the Remnant describe them, as microbots, because A. that's the only mission content where they are relevant, and B. to distinguish them from the *nano*bots of Sayiban infamy.)

Reply:
Then, perhaps it should be `large number`, or specifically say `small volume`.

Reply:
I really think that's too specific and doesn't sound right.

If someone gave you a jar of sand, would you say they gave you a large number or small volume of sand grains?  Only if you were being silly.  Rather, you would say you received a small amount of sand.

Reply:
I'm not sure where you get that idea at all.  If I give you a jar of sand, and a jar of sand-sized microbots, what makes the former uncountable and the latter countable?

Either way it's a moot point.  Countable or not, `amount` is an appropriate descriptor.  Say I gave you a small pile of cash, you could just as well say I gave you a small amount of cash as you could count it and say I gave you $50.12.  Except cash is very easy to count.

Ultimately, this is a stylistic choice, not a grammar issue, and I find amount to be more aesthetically pleasing than number.

Reply:
My two cents here is that it depends on the speaker. If I had a generic crew member talking about them in a high-level general sense, I'd probably have them say. "Cores usually have a small number of microbots." If the same generic crew was talking about a specific core, they'd probably be more likely to say. "This particular core has a small amount of microbots available to repair the ship." If I had Taely or a science officer talking about them, they'd probably opt for the more scientific "This small system core contains a small volume of microbots."

In other words, I find "number" and "amount" and "volume" to be basically synonymous, and differing mainly in how I want to portray the speaker. 

**tldr:** I don't think number vs amount vs volume has sufficient differentiation to be worth protracted discussion on. It's good as is, so I would suggest just leaving it as is.

Reply:
> **tldr:** I don't think number vs amount vs volume has sufficient differentiation to be worth protracted discussion on. It's good as is, so I would suggest just leaving it as is.

Me neither, but I think there is a key component missing to this that gives me a chance to spin lore at y'all -- and that is that there is not a fixed number of microbots in a systems core.  A systems core, as we see with the Shunt-Strike, both controls and *produces* microbots.  Inevitably these little guys are going to get blasted off the hull while they're trying to repair it under fire, or get flung off during high-G manoeuvres, so it has to keep producing them -- and, there lies another key difference between the microbots and nanobots, nanobots self-replicate whereas microbots are produced by a larger device.